### PR TITLE
ssDNA should not show reverseSequence automaticlly 

### PR DESCRIPTION
**Pull request by XingGao-PKI**
With this change and https://github.com/TeselaGen/tg-oss/pull/1
Reversed sequence does not show when drag a ss-DNA file in.
image

### CODE REVIEW

File: src/helperComponents/PropertiesDialog/GenbankView.js [hunk 1]
```diff
@@ -13,6 +13,9 @@ class GenbankView extends React.Component {
   state = {
     fileTypeToView: "genbank"
   };
+
+  select = () => this._input?.select();
```
Comment (XingGao-PKI) [inline src/helperComponents/PropertiesDialog/GenbankView.js:17]: this change fixed the Genbank View selection and also the console warning in dev tool.

**Comment by XingGao-PKI** [comment]
@tnrich could you check this and https://github.com/TeselaGen/tg-oss/pull/1.
Any concerns you can comment here.

**Review by tnrich — APPROVED** [review]

**Comment by tnrich** [comment]
@XingGao-PKI it looks like adding `annotationVisibilityUpdated` to the `isAlreadySpecialEditor` causes translations to not show up on the default sequence and broke a few tests

This code in particular:

image

Broke these tests - 
image

https://cloud.cypress.io/projects/mp89gp/runs/1654/overview?roarHideRunsWithDiffGroupsAndTags=1

I'm going to revert that part of your change from ove. Can you explain why you added that code?

**Comment by XingGao-PKI** [comment]
if you init the editor with a SS-DNA, it will be fine, but if you pull one ss-DNA to ds-DNA borad, we need to update the `reverseSequence`. it's OK you reverted this part, I will check if we have another way to change it.

**Pull request merged**